### PR TITLE
[fix #7240] Add rebranded campaign page

### LINKED ISSUE #7240: Migrate /campaign to a new template
The /campaign pages ('new' and 'thanks') needs to be updated to match the new (Trailhead) page branding in THE /new and /thanks pages.

Tasks

- [ ] Lang file creation
- [ ] Create L10n switching logic
- [ ] Code
- [ ] Test
- [ ] Code review

### PR DESCRIPTION
**Pull request by craigcook**
## Description
This duplicates the trailhead /new template and related files, and will create a new lang file. Locales without the new lang file can fall back to the old template, and eventually we can delete that.

## Issue / Bugzilla link
#7240 

## Testing
http://bedrock-demo-craigcook.oregon-b.moz.works/firefox/campaign/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/campaign/index-trailhead.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    desc=_('And start getting the respect you deserve with our family of privacy-first products.'),
+    class='mzp-has-image mzp-t-dark',
+    include_cta=True,
+    image_url='firefox/new/trailhead/browser-window.svg',
+    heading_level=1
+  ) %}
+    {{ download_firefox(alt_copy=_('Download Now'), locale_in_transition=True, download_location='primary cta') }}
+
+    <ul class="small-links desktop">
+      <li><button class="platform-modal-button js-platform-modal-button hidden" type="button">{{_('Advanced install options & other platforms') }}</button></li>
```
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/firefox/templates/firefox/campaign/index-trailhead.html]: I'm happy to remove and simplify. I'll check with Justin to be sure we're OK to remove this.

**Comment by alexgibson** [comment]
Oh, looks like tests need updating!

File: media/css/mozorg/home/fundraiser.scss [hunk 2]
```diff
@@ -63,7 +63,6 @@ $image-path: '/media/protocol/img';
 .c-fundraiser-options {
     label {
         @include bidi(((margin, 0 $spacing-sm $spacing-md 0, 0 0 $spacing-md $spacing-sm),));
-        @include open-sans;
```
Comment (craigcook) [inline media/css/mozorg/home/fundraiser.scss]: Unrelated to this PR, just a teeny fix since Protocol 7 landed and I figured I'd sneak it in here rather than do another PR.
